tests/dataloader/test_graph_loader.py

```diff
@@ -9,22 +9,32 @@
     "method, kwargs",
     [
         ("knn", {"k": 2}),
+        ("knn", {"k": 2, "percentile": 50}),
+        ("knn", {"k": 2, "max_distance": 50}),
         ("radius", {"max_distance": 50}),
```

Comment:
I added missing test cases. I have just noticed that I forget `radius` with `percentile`, adding it in the next commit.

Reply:
Done.